### DISCUSSION TITLE
Good first issue

### DISCUSSION THREAD

**Discussion by thomashirtz** [EdanToledo/Stoix#79] [General]
Hello,
I saw this project today and I would be quite motivated to contribute to it.
Is there some "good first issue" to tackle or some small changes to the codebase that needs to be done ?
Cheers,
Thomas

**Comment by EdanToledo** [comment]
Hi Thomas, 

Thanks for the interest 🥳. I'm always welcoming contributors and there's always work to be done. A good first issue semi-depends on your proficiency with RL. For example, adding new algorithms even simpler ones are always welcome. Additionally depending on your proficiency with JAX theres the multi-host issue I currently have up. Lastly more boring work but I would like to start adding correct docstrings to all the functions and systems. I would also like to create a docs page like cleanrl has. Someplace where we can place the mathematical formulas underpinning an algorithm. Let me know what you think. If there's anything you think would be a good addition to the library that I haven't thought of, I'm always looking for suggestions and feedback as well.

**Reply by thomashirtz** [reply]
Thank you for your comprehensive reply, I'll start by doing some small modification, then I may ask some more guidance for more advanced features :)
